app.json

```diff
@@ -1,5 +1,13 @@
 {
   "expo": {
-    "sdkVersion": "15.0.0"
+    "sdkVersion": "15.0.0",
+    "name": "Serlo-ABC",
```

Comment:
Andrej said, that he sent you the instructions already for how to build a production version of the app, I will attach it here just for history purposes - basically it just uses Expo utility for that.
But if you'll decide that it's worth to put it in separate `npm` task - we can always do it :)

> I’ve added a configuration file to our repository to build a production version of application using Expo: first one needs to install exp (either globally with `-g` param or locally):
> `npm install exp`
> The run some command with it in order for it to propose to create an account (If you’ve installed it globally *./node_modules/.bin* part shouldn’t be there):
> `./node_modules/.bin/exp start`
> Go through the process of creating an account, and then run:
> `./node_modules/.bin/exp build:android` (`ios`)
> It will take about 10-15 minutes to build an .apk file, you can check the process with:
> `./node_modules/.bin/exp build:status` -> in the end it will write url to the newly generated application file.